notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>I know it's in the comment, but consider adding a title to the table itself</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/CI2023-RC-team2/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Done</p>